<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wlist</class>
 <widget class="QWidget" name="Wlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wlist</string>
  </property>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>49</y>
     <width>39</width>
     <height>258</height>
    </rect>
   </property>
   <property name="sliderPosition">
    <number>8</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnDown">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>308</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnUp">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Label_BtnUp_Bg">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Label_BtnDown_Bg">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>308</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>verticalSlider</zorder>
  <zorder>Label_BtnUp_Bg</zorder>
  <zorder>Label_BtnDown_Bg</zorder>
  <zorder>BtnDown</zorder>
  <zorder>BtnUp</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
